components/Link/index.js

```diff
@@ -109,38 +113,29 @@ const Link = ({
 	onTextChange,
 	kind,
 	placeholder,
+	className,
 }) => {
 	const ref = useRef();
 	const [isLinkOpen, setIsLinkOpen] = useState(false);
 	const [isValid, setIsValid] = useState(false);
 	const { isSelected } = useBlockEditContext();
+	const openPopover = () => setIsLinkOpen(true);
+	const closePopover = () => setIsLinkOpen(false);
+
+	const popoverRef = useOnClickOutside(closePopover);
 
 	const link = {
 		url,
 		opensInNewTab,
 		title: value, // don't allow HTML to display inside the <LinkControl>
 	};
 
-	const blockProps = useBlockProps({
```

Comment:
The useBlockProps hook is really only meant for blocks itself and the props should only be used on the outer most wrapper element. Anything else will lead to undesired behaviors